CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+
+    foundation-2016 -> develop
+
+If a release branch is not active, Foundation 2016 is automatically merged forward to develop, so that bug and security fixes make it into future releases.
+
+    foundation-2016 -> Meridian develop
+
+Foundation 2016 is also merged automatically into the Meridian development codebase, to become part of a future Meridian release. Meridian develop is a very thin set of changes on top of foundation-2016, mostly for branding and a few default configuration changes.
+
+## Where to Merge Code
+
+In short, here is what you should branch from if you are writing new code:
+
+*Critical Security or Bug Fixes:*
+
+    foundation
+
+*Non-Blocker Bug Fixes:*
+
+    foundation-2016
+
+*New Features:*
+
+    develop
+
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic addresses, without explicit permission
+* Other unethical or unprofessional conduct
+* Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions * that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing this project. Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team.
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a project maintainer at [INSERT EMAIL ADDRESS]. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. Maintainers are obligated to maintain confidentiality with regard to the reporter of an incident.
```

Comment:
I created an abuse@opennms.org e-mail address, you should be able to use that.

Thanks for doing this!